packages/playground/website/src/components/browser-chrome/index.tsx

```diff
@@ -43,20 +48,42 @@ export default function BrowserChrome({
 		[css.isHidden]: noticeHidden,
 	});
 
+	/**
+	 * Temporary feature flag to enable the site manager
+	 * while using browser storage.
+	 *
+	 * TODO: Remove this once the site manager supports all storage options.
+	 */
+	const query = new URL(window.location.href).searchParams;
```

Comment:
It's possible to make this more direct like:
```suggestion
	const query = new URLSearchParams(location.search);
```